lib/smack/SmackRep.cpp

```diff
@@ -1012,13 +1017,47 @@ std::string SmackRep::code(llvm::CallInst& ci) {
 
   std::string s = fmt;
   for (unsigned i=1; i<ci.getNumOperands()-1; i++) {
-    const Expr* a = arg(f, i, ci.getOperand(i));
+    Value* a = ci.getOperand(i);
```

Comment:
I guess `a` here represents an argument. Rename this into `arg`.

Reply:
I changed it into `argV` because there's a function called `arg`.